packages/lib-classifier/src/helpers/useJSONData/README.md

```diff
@@ -0,0 +1,17 @@
+# useJSONData
+  
+A custom hook which loads JSON data for a Panoptes subject.
```

Comment:
[minor] While this is true, the description is missing one important detail: `useJSONData` loads JSON data for a Subject, and assumes that Subject that has _exactly one JSON(-ish) URL data._

Specifically:
- A text file "technically" counts as a JSON URL (hence, _JSON-ish_ ) for this function, because the Panoptes CLI has (had?) issues differentiating JSON/TXT MIME types.
- If a Subject has multiple JSON(-ish) URLs, only the first is pulled by useJSONData()

This doesn't matter in current (2018 to 2022) LightCurveViewer (or BarChartViewer, I guess) scenarios, since project owners who use these Viewers know the precise Subject format required. However, I can imagine an edge case where useJSONData doesn't work as described, e.g. when we have a Subject that looks like...

```
subjectWithTextAndJSON.locations = {
  'text/plain': 'actual-boring-plain-text.txt',
  'application/json': 'json-object-for-data-viewer.json'
}
// In this case, the function attempts to load the plain text as JSON, fails at response.json(), and then skips the actual JSON object file.
```

I'm marking this as a minor thing right now (⭐ I think editing the README text to include the caveats above should suffice), because if a future theoretical SuperDataViewer needs to use this function to get a Subject with 10 JSON files and a dozen TXT files, we can adjust the code accordingly when it's less theoretical.

Reply:
Really good point, but I think this can wait until we have real examples of subjects, rather than try to second guess what they might look like now.